### ISSUE TITLE
tootctl search deploy fails with ruby 3.0.1

### ISSUE THREAD

**Issue by mtippmann** [tootsuite/mastodon#16344]
### Expected behaviour

running `RAILS_ENV=production bin/tootctl search deploy` indexes into elasticsearch. 

### Actual behaviour

`RAILS_ENV=production bin/tootctl search deploy` fails with: 

```
Error importing AccountsIndex: wrong number of arguments (given 2, expected 1)                                                              
Error importing AccountsIndex: wrong number of arguments (given 2, expected 1)                                                              
Error importing AccountsIndex: wrong number of arguments (given 2, expected 1)                                                              
Error importing AccountsIndex: wrong number of arguments (given 2, expected 1)
...
``` 

same for other indices 

this looks like it's an issue with ruby 3.0 as discussed here: 
https://rubyreferences.github.io/rubychanges/3.0.html#keyword-arguments-are-now-fully-separated-from-positional-arguments 

but I'm not familiar with ruby - maybe it's just an easy fix - the error is thrown somewhere here between line: 89-136
https://github.com/tootsuite/mastodon/blob/main/lib/mastodon/search_cli.rb according to the exception. 

### Steps to reproduce the problem

- use `rbenv` to install ruby 3.0.1 
- install mastodon 3.4.0 with `.ruby-version` set to 3.0.1 

### Specifications

tag 3.4.0

**Comment by ClearlyClaire** [comment]
Looking into it but I don't have ElasticSearch installed at the moment. Could you remove the `rescue` block on lines 137-138 of `lib/mastodon/search_cli.rb` and try again to get a detailed traceback?

**Comment by ClearlyClaire** [comment]
It should be fixed by #16346 but I haven't got to setting up ElasticSearch yet, so it would be helpful if you could try and confirm!

**Comment by mtippmann** [comment]
thanks for quick fix! after switching back to ruby 3.0.1 and running the reindex again the exception is gone and it works as with ruby 2.7.3!
